[Rev 1 Docs] [Design] [SD] Add Description to Mockups (#728)
arkinmodi/project-sayyara: docs/Design/SystDesign/SystDes.tex

% 	needed. Drawings, Sketches, Figma}

\subsection{Home Page}

> The Home Page features all the auto shops that match the search criteria. Users can search by shop

Reviewer: ```suggestion
The Home Page features all the auto repair shops that match the search criteria. Users can search by shop
```

Reviewer: Add after first sentence:
"By default, when no search criteria is entered, the Home Page will show all shops in a paginated list view.

\end{figure}

\subsection{Manage Shop Employees}

> The Manage Employees page is where shop owners can manage their employees by searching for their

Reviewer: ```suggestion
The Manage Shop Employees page is where shop owners and employee can view a tabular view of all shop employees. Shop owners can manage their employees by searching for their
```

employee, viewing employee details such as name, phone number, email, etc. Shop owners can also

> suspend employees from this page.

Reviewer: ```suggestion
employee, suspend employees, and view employee details such as name, phone number, email, etc.
```

Reviewer: Add:
Employees have limited view-only permissions.

\end{figure}

\subsection{Shop Profile}
The Shop Profile page is where shop owners can view their shop profile which vehicle owners can

> view. On this page, shop owners can edit their hours of operation.

Reviewer: ```suggestion
The Shop Profile page contains information about a shop such as location, contact information, hours of operation, and services offered. Vehicle owners, employees, and shop owners can view, however, only the shop employee has edit permission to edit the hours of operation.
```

\end{figure}

\subsection{Shop Owner/Employee Registration}
These popups show the sign up form for shop owners and employees. The sign up has multiple steps
which the user can access once completing the previous steps. When all fields are entered and

> validated, the user can successfully sign up.

Reviewer: ```suggestion
These popups show the sign up form for shop owners and employees. The sign up has multiple steps
which the user can access once completing the previous steps. When all fields are entered and
validated, the user can successfully sign up. Depending on if the user is signing up as an employee or shop owner, they will be directed to different steps containing relevant fields required for registration of that specific role.
```

\end{figure}

\subsection{Vehicle Owner Registration}

> These popups show the sign up for vehicle owners. The form includes information about the vehicle

Reviewer: ```suggestion
This shows the sign up process for vehicle owners. The form includes information about the vehicle
```

\end{figure}

\subsubsection{Quotes}
The Quotes page displays any chats the vehicle owner has had with shop owners to receive a quote.

> Vehicle owners can send chat messages to shops using this page.

Reviewer: ```suggestion
The Quotes page displays any chats the vehicle owner has had with shop owners to receive a quote.
Vehicle owners can send chat messages to shops using this page. Users can also query and search for a specific chat using the search bar.
```

\end{figure}

\subsection{Vehicle Owner Create Appointment}
These pages are popups that will be displayed when a vehicle owner wants to create an appointment
at shop. The shop availabilities will be displayed according to their hours of operations and any

> hours that currently do not have a service requested, scheduled, or in-progress.

Reviewer: ```suggestion
The popup to create an appointment contains various steps. The grey fields represent pre-filled fields that are read-only. These fields are populated using vehicle metadata associated with the user and service metadata associate with the service. The shop availabilities will be displayed according to their hours of operations and any hours that currently do not have a service requested, scheduled, or in-progress.
```

\subsection{Shop Owner/Employee Dashboard}
\subsubsection{Quotes Requests}
The Quotes page displays any chats that have been initiated by vehicle owners. Shop owners and

> employees can send chat messages to vehicle owners on this page.

Reviewer: ```suggestion
The Quotes page displays any chats that have been initiated by vehicle owners. Shop owners and
employees can send chat messages to vehicle owners on this page. Users can also query and search for a specific chat using the search bar. Shop owners can invite an employee to schedule an appointment once they are ready.
```

	\caption{Shop Owner/Employee Dashboard \textemdash{} Quotes Requests \textemdash{} Invitation (Desktop)}
\end{figure}

\subsubsection{Service Requests \textemdash{} Requested} The Requested Services are shown on this page. By clicking on an appointment, the work

> order for the appointment will be displayed.

Reviewer: ```suggestion
\subsubsection{Service Requests \textemdash{} Requested} The Requested Services are shown on this page. By clicking on an appointment, the work order for the appointment will be displayed. Requested Services can be rejected or accepted. Accepting an appointment will automatically reject all conflicting appointments.
```
